sql_generators/usage_reporting/templates/usage_reporting_clients_daily_v1.schema.yaml

```diff
@@ -0,0 +1,90 @@
+{{ header }}
+fields:
+- mode: NULLABLE
+  name: submission_date
+  type: DATE
+  description: |
+    Logical date used for processing and paritioning.
+
+- mode: NULLABLE
+  name: usage_profile_id
+  type: STRING
+  description:
+
+- mode: NULLABLE
+  name: first_run_date
+  type: DATE
+  description: |
+    The date of the first run of the application.
+
+- mode: NULLABLE
+  name: normalized_channel
+  type: STRING
+  description: |
+    The channel the application is being distributed on.
+
+- mode: NULLABLE
+  name: normalized_country_code
+  type: STRING
+  description: |
+    Country code
+
+- mode: NULLABLE
+  name: normalized_os
+  type: STRING
+  description: |
+    The name of the operating system.
+
+- mode: NULLABLE
+  name: normalized_os_version
+  type: STRING
+  description: |
+    The user-visible version of the operating system (e.g. "1.2.3").
+    If the version detection fails, this metric gets set to Unknown.
+
+- mode: NULLABLE
+  name: app_build
+  type: STRING
+  description: |
+    The build identifier generated by the CI system (e.g. "1234/A").
+    If the value was not provided through configuration, this metric gets set to Unknown.
+
+- mode: NULLABLE
+  name: app_display_version
+  type: STRING
+  description: |
+    The user visible version string (e.g. "1.0.3").
+    If the value was not provided through configuration, this metric gets set to Unknown.
+
+- mode: NULLABLE
+  name: distribution_id
+  type: STRING
+  description: |
+    A string containing the distribution identifier. This was used to identify installs
+    from Mozilla Online, but now also identifies partnership deal distributions.
+
+- mode: NULLABLE
+  name: is_default_browser
+  type: BOOLEAN
+  description: |
+    Is Firefox the default browser.
+
+- mode: NULLABLE
+  name: reason
+  type: STRING
+  description: |
+    The optional reason the ping was submitted. The specific values for reason are specific to each ping,
+    and are documented in the ping's pings.yaml file.
+
+- mode: NULLABLE
+  name: is_active
+  type: BOOLEAN
+  description: |
+    A flag field indicating whether the specific client was active.
+
+# - mode: NULLABLE
```

Comment:
confirmed that you can remove this. not present in the ping and not approved to be.

Reply:
by this you mean `locale`?

Reply:
removing.